src/routes/index.js

```diff
@@ -23,7 +23,13 @@ export default function getRoutes(renderRequestMap, apiProxyFn$ = apiProxy$) {
 		method: 'GET',
 		path: '/_ah/{param*}',
 		config: { auth: false },
-		handler: (request, reply) => reply('OK'),
+		handler: (request, reply) => {
+			if (request.params.checkType === 'error') {
```

Comment:
yeah, you should be able to make a direct copy of [this integration test](https://github.com/meetup/meetup-web-platform/blob/master/tests/integration/render.int.js#L211-L228) and just change the request `url` to the expected-to-fail url. Probably best to put the integration test into a new `tests/integration/gaeRoute.int.js` script, but adding it to `render.int.js` is fine.